controllers/articles.controller.js

```diff
@@ -1,9 +1,32 @@
-const readArticles = require('../models/articles.model')
+const { readArticles, getArticleById, readComments }  = require('../models/articles.model');
 
 const getArticles = (request, response) => {
     readArticles().then((articles) => {
-        response.status(200).send({articles})
+        const articlesWithoutBody = articles.map((article) => {
+            const { body, ...articleWithoutBody } = article;
```

Comment:
Instead of mapping over and removing the body property, we can simply not select the body column in the model SQL statement